scripts/update_localization.rb

```diff
@@ -73,6 +76,26 @@
                       "messages#{@options[:pot_suffix]}"
     end
   end
+
+  # Remove trailing space
+  info "Removing trailing space..."
+
+  LOCALES.each do |locale|
+    target = locale + @options[:po_suffix]
+    trimmedtext = ""
+    changed = false
+    File.readlines(target, :encoding => "utf-8").each do |line|
+      trimmed = line.rstrip() + "\n"
+      if line != trimmed then changed = true; end
+      trimmedtext += trimmed
+    end
+
+    # Babel generates identically formatted files. So when the first one doesn't have trailing space, nor will the others.
+    if !changed then break; end
```

Comment:
Pretty sure this isn't what rubocop wants so could you run rubocop auto fix on this file? So I don't have to.